eng/Packages.Data.props

```diff
@@ -194,8 +194,9 @@
     <PackageReference Update="NUnit" Version="3.13.2" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Provisioning.CloudMachine'))">
-    <PackageReference Update="Microsoft.Extensions.Hosting.Abstractions" Version="9.0.0" />
+  <ItemGroup Condition="$(MSBuildProjectName.Contains('CloudMachine'))">
+    <PackageReference Update="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0"/>
```

Comment:
FYI - I'm about to bump this for the repo, though you may need to set `IsExtensionClientLibrary` if you're not already do so - or we'd need to promote this to an unconstrained set.

Likewise, we should look at moving hosting abstractions to the test area and/or the extensions area.    We currently have Hosting but not Hosting.Abstractions.

Reply:
I synced this morning and it does not seem like you did the bump. Since you signed off on the PR, I am assuming I can merge and once the bump happens, we will adjust.

Reply:
> I synced this morning and it does not seem like you did the bump. Since you signed off on the PR, I am assuming I can merge and once the bump happens, we will adjust.

Yup, good with the merge.   I tripped over an unexpected failure when doing test runs last night, so I'll look at trimming this back as part of that work.